test/unit_tests/sample_scheme_files/duplicate_dyn_const.F90

```diff
@@ -0,0 +1,96 @@
+! Test parameterization with no vertical level
```

Comment:
Can you update the description?
Also, is this test exercising duplicates?

Reply:
good point - updated the description. It's exercising duplicate dynamic constituent routine _names_